### ISSUE TITLE
Failed payment when I created an invoice with my LND node through Alby

### ISSUE THREAD

**Issue by MoritzKa** [getAlby/lightning-browser-extension#868]
**Describe the bug**
I want to receive 1 sat on my LND node. The LND node is connected with the Alby extension. I create an invoice with the extension for my LND node and pass it on to the Bitcoin Beach Wallet. After I confirmed the payment I get: "there was an error. Unable to find a route for payment"
My node has one channel (with the Alby node) and enough inbound capacity. 
However, the invoice generated by my Zeus app (also connected to my LND node) also does not result in a successful payment from Bitcoin Beach wallet.

**To Reproduce**
1. Connect LND node with Alby
2. Click on LND account in Alby and generate an invoice
3. Pay that invoice from another account/app etc.
4. See error "Unable to find a route for payment"

**Expected behavior**
a successful payment

**Information About Alby**
- Alby Version 1.10.0
- Alby installed through the browser stores
- Wallet connected with Alby LND

**Screenshots (If Any)**
Add screenshots to help explain your problem.

**Device Information [optional]:**
 - OS: Windows
 - Browser: Chrome

**Additional context**
Problem reported by another user: https://feedback.getalby.com/bugs-errors--1/posts/payment-failed

**Are you working on this issue? (Yes/No)**
No

**Comment by PrinzMonty** [comment]
The same problem for me and sometimes I become an Error "Faild to fetch".

With my Lnbits (same node) it works fine :)


![image](https://user-images.githubusercontent.com/4109582/167298907-27d995c5-4d3e-4014-8544-fce7aca81c7c.png)

**Comment by bumi** [comment]
@PrinzMonty did you get this today? what did you try to pay when you got this error?

**Comment by PrinzMonty** [comment]
I test with very small amounts of Sats 20 - 50 Sats.

**Comment by bumi** [comment]
If it was towards an Alby lightning address or something hosted on getalby.com then we had some DDoS attack today which caused some problems. 
Right now everything looks stable again, our defense is holding up. :) Can you test it again, to see if your issue is related?
https://twitter.com/getAlby/status/1523220562851508224

**Comment by PrinzMonty** [comment]
Nope. Same error ... I get the error for days, but most of the time it worked.

But my receipt problem with my LND node I have all the time.

Maybe its an setting problem on my site, but I don't understand why it works with LNbits on the same node?!

**Comment by bumi** [comment]
I think Alby currently does not include "private" route hints. Others maybe do this. 

We should investigate how Zeus, Lnbits, etc. create invoices and if they include those by default. (or if this is a user option?)

**Comment by MoritzKa** [comment]
Feedback from a user
Sep 29, 2022
When I connect my Raspiblitz via Tor or Local IP, I cannot receive any payments. The transaction always fails.

The funny thing is that I also run a BTCPayServer and Lnbits on the same node and when I include those I can receive payments without any problems :D

**Comment by wksantiago** [comment]
Getting same error when trying to send payment to Bitrefill

**Comment by bumi** [comment]
@wksantiago what is your exact error? and what account/node do you have connected?

**Comment by wksantiago** [comment]
When I try to paste and  pay a Lightning invoice I get this:

"Error: Failed to fetch"

**Comment by wksantiago** [comment]
@bumi I have an account in the extension with a few sats not connected to a node

**Comment by wksantiago** [comment]
@bumi My bad I did have two more connected nodes and those had sats, but my hot wallet in the extension had 0 sats.

After removing the two nodes and just leaving the Alby wallet I then added some sats to it, and then it worked.
